metadata/junit/junit.go

```diff
@@ -130,10 +131,12 @@ func (r *Result) SetProperty(name, value string) {
 
 // Message extracts the message for the junit test case.
 //
-// Will use the first non-empty <failure/>, <skipped/>, <system-err/>, <system-out/> value.
+// Will use the first non-empty <error/>, <failure/>, <skipped/>, <system-err/>, <system-out/> value.
 func (r Result) Message(max int) string {
 	var msg string
 	switch {
+	case r.Errored != nil && *r.Errored != "":
```

Comment:
Add a corresponding test case to TestMessage().

Reply:
Done